src/WebUI/Utils.ts

```diff
@@ -82,11 +84,13 @@ export class Utils {
             return {} as IMetadataAnnotationPipeline;
         }
 
+        const runName = !!annotations[pipelineRunIdAnnotationKey] ? annotations[pipelineRunIdAnnotationKey] : annotations[newPipelineRunIdAnnotationKey];

```

Comment:
we should depend upon new values, i mean check for newPipelineRunIdAnnotationKey and take the value.

you can even do:
annotations[newPipelineRunIdAnnotationKey] || annotations[pipelineRunIdAnnotationKey];

same with other key